### PR TITLE
 Ambassador program

### PR DESCRIPTION
**Pull request by midegdugarova**

### CODE REVIEW

File: docs/ambassadors.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+id: ambassadors
+title: Polkadot Ambassador Programme
+sidebar_label: Polkadot Ambassador Programme
+---
+
+The Polkadot Ambassador Programme is an initiative run by Web3Foundation to support contributors to
```
Comment (joepetrowski) [inline docs/ambassadors.md]:
```suggestion
The Polkadot Ambassador Programme is an initiative run by Web3 Foundation to support contributors to
```

File: docs/ambassadors.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+the Polkadot community and ecosystem (Ambassadors).
+
+## How it works
+
+If you are enthusiastic about Polkadot and Web3 and are willing to help the community grow you can
+join the program and receive funding for hosting events, access to communication channels of
+core-team members, expanded access to Polkadot events.
```
Comment (joepetrowski) [inline docs/ambassadors.md]:
```suggestion
core team members, and expanded access to Polkadot events.  
```

File: docs/ambassadors.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+The Polkadot Ambassador Programme is an initiative run by Web3Foundation to support contributors to
+the Polkadot community and ecosystem (Ambassadors).
+
+## How it works
+
+If you are enthusiastic about Polkadot and Web3 and are willing to help the community grow you can
+join the program and receive funding for hosting events, access to communication channels of
+core-team members, expanded access to Polkadot events.  
+There are many ways you can be involved in expanding Polkadot’s ecosystem, for example by:
```
Comment (joepetrowski) [inline docs/ambassadors.md:16]:
```suggestion

There are many ways you can be involved in expanding Polkadot’s ecosystem, for example by:
```

**Review by joepetrowski — COMMENTED** [review]
Inconsistent use of program/programme (e.g. L13, L31)

**Review by laboon — APPROVED** [review]

**Pull request merged**